src/NuGet.Core/NuGet.Commands/ClientCertificatesCommand/ListClientCertRunner.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.Commands
+{
+    public class ListClientCertRunner
+    {
+        public static void Run(ListClientCertArgs args, Func<ILogger> getLogger)
+        {
+            args.Validate();
+            var settings = RunnerHelper.GetSettings(args.Configfile);
+            var clientCertificateProvider = new ClientCertificateProvider(settings);
+
+            var items = clientCertificateProvider.GetClientCertificates();
+
+            if (!items.Any())
+            {
+                getLogger().LogInformation(Strings.NoClientCertificates);
+                return;
+            }
+
+            var clientCertificatesLogs = new List<LogMessage>();
+
+            getLogger().LogInformation(Strings.RegsiteredClientCertificates);
+            getLogger().LogInformation(Environment.NewLine);
+
+            for (var i = 0; i < items.Count; i++)
+            {
+                var item = items[i];
+
+                var builder = new StringBuilder();
+
+                var index = $" {i + 1,-6}.";
```

Comment:
When I run `.\artifacts\VS15\nuget.exe client-certs`, I get:

```
Registered client certificates:


 1     .vsssdk [fileCert]
      File path: .\so.pfx
      Password: ****
      Certificate: AFD2801DEBD506D53634EAFD0F09CAE93107C198

 2     .dotnet-core [fileCert]
      File path: .\tw.pfx
      Password: ****
      Certificate: 763AFA1435BDFEE955CEA3A66973F1088186B1C7

```

I imagine that the dot is for separating the number. Is that correct? Also, I prefer the same padding level as the `nuget.exe sources` command, which is 6 spaces

Define static value that defines padding width:
```csharp
        private const int PaddingWidth = 6;
```

Move the padding width calculation for the number out of the for loop

```csharp
            int paddingWithNumber = PaddingWidth - 1;
```

Then, create the string for the number

```suggestion
                var numDot = $"{i}.";
                var index = $"{numDot, paddingWithNumber} ";
```